src/Widgets/AppContextMenu.vala

```diff
@@ -62,6 +73,24 @@ public class Slingshot.AppContextMenu : Gtk.Menu {
             });
         }
 
+        if (switcheroo_control != null && switcheroo_control.has_dual_gpu) {
+            bool prefers_non_default_gpu = app_info.get_boolean ("PrefersNonDefaultGPU");
+
+            string label = prefers_non_default_gpu ?
+                _("Launch using Integrated Graphics Card") :
+                _("Launch using Discrete Graphics Card");
```

Comment:
I think we usually use the term “open” for apps, instead of “launch”

```suggestion
                _("Open using Integrated Graphics Card") :
                _("Open using Discrete Graphics Card");
```

I also wonder if there's a more concise way to word these, as this makes the menu pretty long.

Reply:
Some alternatives:

- Open with NVIDIA GPU / Open with Intel GPU
- Open with Discrete GPU / Open with Integrated GPU

Reply:
@jhaygood86 yeah I think I'm leaning towards "Open with NVIDIA Graphics"/"Open with Intel graphics" to start, making sure we actually get the GPU manufacturer name in the case of Intel/AMD or AMD/NVIDIA systems. :wink: